### PR TITLE
Fix permission error on double click on submit determination

### LINKED ISSUE #3342: [Bug] You do not have permission to create that determination
### Description

Users are getting this error message when they're sending out determinations. 

Screenshot 2023-03-20 1 17 33 PM (1)

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes #3342 

This PR contains changes only for determination. We will be implementing the same approach for other forms as well.

### CODE REVIEW

File: hypha/apply/determinations/templates/determinations/base_determination_form.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h5>{% trans "For" %} <a href="{% url "funds:submissions:detail" submission.id %
     {% block determination_information %}
     {% endblock %}
 
-    <form class="form form--with-p-tags" action="" method="post">
+    <form class="form form--with-p-tags" action="" method="post" x-data="{ submitButtonDisabled: false }" x-on:submit="submitButtonDisabled = true">
```
Comment (theskumar) [inline hypha/apply/determinations/templates/determinations/base_determination_form.html]:
```suggestion
    <form class="form form--with-p-tags" action="" method="post" x-data="{ isFormSubmitting: false }" x-on:submit="isFormSubmitting = true">
```

File: hypha/apply/determinations/templates/determinations/base_determination_form.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h2>{{ value }}</h2>
             {% if form.draft_button_name %}
                 <button class="button button--submit button--top-space button--white" type="submit" name="{{ form.draft_button_name }}" formnovalidate>{% trans "Save draft" %}</button>
             {% endif %}
-            <button class="button button--submit button--top-space button--primary" type="submit" name="submit">{% trans "Submit" %}</button>
+            <button class="button button--submit button--top-space button--primary" x-bind:disabled="submitButtonDisabled" type="submit" name="submit">{% trans "Submit" %}</button>
```
Comment (theskumar) [inline hypha/apply/determinations/templates/determinations/base_determination_form.html]:
```suggestion
            <button class="button button--submit button--top-space button--primary" :disabled="isFormSubmitting" type="submit" name="submit">{% trans "Submit" %}</button>
```

**Review by theskumar — CHANGES_REQUESTED** [review]
Model the variables as the current state rather the side effects of the those states. 

Alphine is primarily used for state management of the UIs

**Comment by frjo** [comment]
Neat solution!

**Pull request merged**